tests/decoder_compliance.rs

```diff
@@ -2,10 +2,7 @@ fn main() {
     let decoder = Decoder;
     let mut harness = toml_test_harness::DecoderHarness::new(decoder);
     harness
-        .ignore([
-            "valid/inline-table/key-dotted.toml",
-            "valid/string/multiline-quotes.toml",
-        ])
+        .ignore(["valid/string/multiline-quotes.toml"])
```

Comment:
nice to see we're getting there :)

Reply:
The checklist in #91 still has some big things left, but progress is still good.